### ISSUE TITLE
Command Timeout in bluetooth_info_test.py with --ble flag

### ISSUE THREAD

**Issue by irvinec** [irvinec/SpheroPy#3]
Consistently getting a command timeout issue on my SPRK+ when running bluetooth_info_test.py --ble.

**Comment by josephcappadona** [comment]
Any update on this? I am trying to use this library to interface with an R2D2 Sphero. It says "Connected to Sphero", but all of the actions time out. Does this issue occur with all of the BTLE devices, or is the R2D2 just not compatible?

**Comment by irvinec** [comment]
Hi @josephcappadona,

Thanks for your interest in SpheroPy and in making it better!
This particular issue is specific to the `bluetooth_info_test`. It sounds like you are hitting a different issue.
I do most of my testing on a Sphero SPRK+ which uses BLE and the rest of the commands **should** work consistently. I have not tested with the R2D2 and it is possible it is not compatible, but I'll see if I can find out more.

Some info that would help me diagnose your issue:
1) Which OS are you using? (Windows 10, Ubuntu 16.04, ...)
2) Which bluetooth interface are you using? (winble, pygatt, ...)
3) Which tests/commands have you tried that are failing with a timeout? Having code or knowing which of my tests reproduce the issue would be best.

Thanks again!

**Comment by irvinec** [comment]
Hi @josephcappadona,

I've discovered that currently SpheroPy will not work with the R2D2, Sphero BOLT, and a few other Sphero robots.

After looking at libraries that support the R2D2, I realize that SpheroPy only uses the "older", documented packet format. The R2D2 uses a different set of bluetooth characteristics and a different packet format that is largely undocumented. However, there seems to be some great work that others have done to reverse-engineer the characteristics and the packet formats necessary to work with the "newer" Sphero robots.

I'd like to add support for these, but unfortunately it won't be in the near future. I am always open to contributions though.

Here are the libraries I referenced in-case they help you (and to give them credit).
https://github.com/MProx/Sphero_mini
https://github.com/igbopie/spherov2.js

Thanks very much for bringing this limitation to my attention! I'll add a "supported" list to my README.
